desktop/cmp/appOption/SizingModeAppOption.ts

```diff
@@ -4,19 +4,29 @@
  *
  * Copyright © 2022 Extremely Heavy Industries Inc.
  */
-import {SizingMode, XH} from '@xh/hoist/core';
+import {AppOptionSpec, SizingMode, XH} from '@xh/hoist/core';
 import {button} from '@xh/hoist/desktop/cmp/button';
 import {buttonGroupInput} from '@xh/hoist/desktop/cmp/input';
 import '@xh/hoist/desktop/register';
 import {startCase, values} from 'lodash';
 
+interface SizingModeAppOptionSpec {
+    /** Supported SizingModes */
+    modes?: SizingMode[],
+    /** Props for nested FormField - todo: replace with FormFieldProps */
+    formFieldProps?: Record<string, any>,
```

Comment:
Could probably use `HoistProps` for a placeholder generally in this situation --- would not be the worst thing in the world if it was merged that way...